cli/tests/unit/tty_color_test.ts

```diff
@@ -24,3 +24,32 @@ Deno.test(
     assertEquals(output, "false\n");
   },
 );
+
+Deno.test(
+  { permissions: { run: true, read: true } },
+  async function denoNoColorTrueEmptyVar() {
+    const { stdout } = await new Deno.Command(Deno.execPath(), {
+      args: ["eval", "console.log(Deno.noColor)"],
+      env: {
+        // https://no-color.org/ -- should not be true when empty
+        NO_COLOR: "",
+      },
+    }).output();
+    const output = new TextDecoder().decode(stdout);
+    assertEquals(output, "false\n");
+  },
+);
+
+Deno.test(
+  { permissions: { run: true, read: true } },
+  async function denoNoColorTrueEmptyVar() {
+    const { stdout } = await new Deno.Command(Deno.execPath(), {
+      args: ["eval", "console.log(Deno.noColor)"],
+      env: {
+        NO_COLOR: "1",
```

Comment:
**question:** Does `NO_COLOR=0` also turn on noColor? What's the usual way env flags work with regards to this?

Reply:
According to this https://no-color.org/

> Command-line software which adds ANSI color to its output by default should check for a NO_COLOR environment variable that, when present and not an empty string (regardless of its value), prevents the addition of ANSI color.

We don't necessarily have to follow that though...

Reply:
This seems like an excellent "standard" to follow. Thank you!